src/main/java/domain/Cars.java

```diff
@@ -0,0 +1,79 @@
+package domain;
+
+import exception.InvalidCarNameException;
+import exception.MethodNotAllowedException;
+import input.RacingGameInputAsker;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class Cars {
+    private Optional<List<Car>> cars;
+    public Cars(String userInput) {
+        try {
+            List<String> carNames = parse(userInput);
+            checkValidCarNames(carNames);
+            List<Car> cars = carNames.stream()
+                    .map(carName -> new Car(carName))
```

Comment:
```suggestion
                    .map(Car::new)
```
메소드 레퍼런스를 활용하는 것은 어떻게 생각하나요? 
IntelliJ 를 사용할 때 노란색으로 표시되는 경우가 있는데 확인 해보시면 좋을거 같아요. 항상 베스트 프랙티스는 아니지만 더 나은 방법을 추천해주는데 이 경우에도 메소드 레퍼런스 사용을 추천하네요.